mujoco_warp/_src/io_test.py

```diff
@@ -567,6 +572,23 @@ def test_contact_sensor_maxmatch(self):
 
     self.assertEqual(m.opt.contact_sensor_maxmatch, 5)
 
+  def test_make_data_eq_type(self):
+    mjm, _, _, _ = test_data.fixture(path="constraints.xml")
+
+    d = mjwarp.make_data(mjm)
```

Comment:
The fix seems to be in `put_data` but here you are calling `make_data` - this is not exercising your fix.